src/search.ts

```diff
@@ -34,16 +35,17 @@ export const webviewHtml: (
         var arr = [...searchImages];
 
         // event handler that sends back to the extension the url of the clicked gif
-        const sendUrl = event => {
-          const url = event.target.getAttribute('src');
+        const sendGif = event => {
+					const url = event.target.getAttribute('src');
+					const label = event.target.getAttribute('alt');
```

Comment:
weird indentation here

Reply:
I dont know if this is because of the huge string, but the indentation looks ok on my computer :/